.ci/jenkins/Jenkinsfile.prod.nightly

```diff
@@ -29,7 +29,7 @@ pipeline{
           script {
             // Fetch versions from pom if not provided
             env.PRODUCT_VERSION = "${PRODUCT_VERSION ?: parseVersionFromPom('kiegroup/drools-ansible-rulebook-integration')}"
-            env.DROOLS_PRODUCT_VERSION = "${DROOLS_PRODUCT_VERSION ?: parseVersionFromPom('kiegroup/drools')}"
```

Comment:
It seems the Drools branch 8.38.x needs to be updated together with the .ci/project-dependencies.yaml - like it was done in PR https://github.com/kiegroup/drools-ansible-rulebook-integration/pull/32. In case of a "non main" nightly.

Maybe add a comment in .ci/project-dependencies.yaml saying that we have to also update .ci/jenkins/Jenkinsfile.prod.nightly if we are doing nightly for a "non main" branch.

Reply:
I agree and comment added